cli/install.go

```diff
@@ -105,7 +105,7 @@ func newCmdUninstallWithHelm() *cobra.Command {
 			params.HelmReleaseName = helmReleaseName
 			ctx := context.Background()
 
-			cc, err := check.NewConnectivityTest(k8sClient, check.Parameters{
+			cc, err := check.NewConnectivityTest(0, k8sClient, check.Parameters{
```

Comment:
this won't work if `--test-concurrency` was set to a value greater than 1.

i don't like the fact `install` depends on `connectivity`. let me think about how to decouple these 2 more cleanly 💭

Reply:
let's start with this refactoring: https://github.com/cilium/cilium-cli/pull/2498

after this i'll make the change to delete test namespaces by prefix.

Reply:
https://github.com/cilium/cilium-cli/pull/2503 to delete test namespaces by prefix

Reply:
forget https://github.com/cilium/cilium-cli/pull/2503, you can rebase now 🚀

Reply:
Thank you, Michi!